compiler/rustc_codegen_ssa/src/back/linker.rs

```diff
@@ -223,38 +231,12 @@ pub struct GccLinker<'a> {
 }
 
 impl<'a> GccLinker<'a> {
-    /// Passes an argument directly to the linker.
-    ///
-    /// When the linker is not ld-like such as when using a compiler as a linker, the argument is
-    /// prepended by `-Wl,`.
-    fn linker_arg(&mut self, arg: impl AsRef<OsStr>) -> &mut Self {
-        self.linker_args(&[arg]);
-        self
+    fn linker_arg_(&mut self, arg: impl AsRef<OsStr>) {
```

Comment:
```suggestion
    fn linker_arg(&mut self, arg: impl AsRef<OsStr>) {
```
I didn't notice there was a name conflict.
It should be possible to just keep the old names, inherent methods will shadow methods from the trait anyway.